src/assets.ts

```diff
@@ -442,11 +449,27 @@ function addLaunchJsonIfNecessary(generator: AssetGenerator, operations: Operati
             return resolve();
         }
 
+        // Read existing launch configuration
+        const launchConfigs = vscode.workspace.getConfiguration('launch');
+        let existingLaunchConfigs = launchConfigs.get<{}[]>('configurations');
+
         const isWebProject = generator.hasWebServerDependency();
-        const launchJson: string = generator.createLaunchJson(isWebProject);
+        let launchJson: string = generator.createLaunchJson(isWebProject);
+
+        if (existingLaunchConfigs) {
+            let existingLaunchConfigsString = JSON.stringify(existingLaunchConfigs, null, '    ');
+            const lastBracket = launchJson.lastIndexOf(']');
+            const lastBracketInExistingConfig = existingLaunchConfigsString.lastIndexOf(']');
+            const firstBracketInExistingConfig = existingLaunchConfigsString.lastIndexOf('[');
```

Comment:
On this line, don't you want `existingLaunchConfigsString.indexOf('[');` instead of `lastIndexOf`?

Reply:
Indeed. My mistake. It did work as there was only one occurrence of `[`. Thanks for catching. Let me fix this